cli/cmd/stat.go

```diff
@@ -447,18 +530,25 @@ func namespaceName(resourceType string, key string) (string, string) {
 
 // Using pointers where the value is NA and the corresponding json is null
 type jsonStats struct {
-	Namespace      string   `json:"namespace"`
-	Kind           string   `json:"kind"`
-	Name           string   `json:"name"`
-	Meshed         string   `json:"meshed"`
-	Success        *float64 `json:"success"`
-	Rps            *float64 `json:"rps"`
-	LatencyMSp50   *uint64  `json:"latency_ms_p50"`
-	LatencyMSp95   *uint64  `json:"latency_ms_p95"`
-	LatencyMSp99   *uint64  `json:"latency_ms_p99"`
-	TCPConnections *uint64  `json:"tcp_open_connections"`
-	TCPReadBytes   *float64 `json:"tcp_read_bytes_rate"`
-	TCPWriteBytes  *float64 `json:"tcp_write_bytes_rate"`
+	Namespace      string       `json:"namespace"`
+	Kind           string       `json:"kind"`
+	Name           string       `json:"name"`
+	Meshed         string       `json:"meshed"`
+	Success        *float64     `json:"success"`
+	Rps            *float64     `json:"rps"`
+	LatencyMSp50   *uint64      `json:"latency_ms_p50"`
+	LatencyMSp95   *uint64      `json:"latency_ms_p95"`
+	LatencyMSp99   *uint64      `json:"latency_ms_p99"`
+	TCPConnections *uint64      `json:"tcp_open_connections"`
+	TCPReadBytes   *float64     `json:"tcp_read_bytes_rate"`
+	TCPWriteBytes  *float64     `json:"tcp_write_bytes_rate"`
+	TsStats        *jsonTsStats `json:"ts_stats"`
+}
+
+type jsonTsStats struct {
+	Apex     string `json:"apex"`
+	LeafName string `json:"leaf_name"`
```

Comment:
Using `service` and `backend` would be consistent with the TS CRD but the downside is that both of those words have many different meanings and this could be confusing for users. 
Either way, it may help to be more explicit with the names. WDYT?

Using the TS CRD terms:
```
root_service
   backends
       service_name
       service_weight
```
Using the current terms:
```
apex_service
   leaf_services
       leaf_name
       leaf_weight
```